bin/pycbc_live

```diff
@@ -354,6 +354,15 @@ class LiveEventManager(object):
                     cmd += '--enable-gracedb-upload '
 
                 cmd += '--cores {}'.format(self.fu_cores)
+                if args.processing_scheme:
+                    # we will use the cores for multiple workers of the
+                    # optimization routine, so we force the processing scheme
+                    # to a single core here.  This may be enforcing some
+                    # assumptions about the optimal way to do FFTs on the
+                    # machine.  Do we need more flexibility, and is there a
+                    # simple way to have it?
+                    opt_scheme = args.processing_scheme.split(':')[0]
+                    cmd += '--processing-scheme {}:1'.format(opt_scheme)
```

Comment:
It's worth pointing out that the dominant cost of optimal SNR is waveform generation, not FFTs, so it's more about having an efficient way of generating lots of waveforms.

Reply:
Ah, of course. I will that that note to the comment.